smallrye-reactive-messaging-kafka/src/test/java/io/smallrye/reactive/messaging/kafka/KafkaTestBase.java

```diff
@@ -66,7 +66,9 @@ public void setup() {
 
     @After
     public void tearDown() {
-        vertx.close();
+        if (vertx != null) {
+            vertx.close();
```

Comment:
we should wait until the vert.x instance is closed if we want to be totally correct.